samples/cpp/benchmark_app/infer_request_wrap.hpp

```diff
@@ -42,7 +54,16 @@ class InferReqWrap final {
           outputClBuffer() {
         _request.set_callback([&](const std::exception_ptr& ptr) {
             _endTime = Time::now();
-            _callbackQueue(_id, _lat_group_id, get_execution_time_in_milliseconds(), ptr);
+            const auto& compile_model = _request.get_compiled_model();
+            const auto& outputs = compile_model.outputs();
+            auto output_size = outputs.size();
+            InferenceResult inference_result;
+            for (size_t i = 0; i < output_size; i++) {
+                const auto& tensor = _request.get_output_tensor(i);
+                inference_result.output_tensors.emplace_back(tensor);
+            }
+            inference_result.input_images = get_input_image_name();
```

Comment:
Calling this code on each callback can lead to performance degradation even for cases when dumping output data is not required.

Reply:
I will add conditional judgement to reduce the impact on performance.